source/profiles.c

```diff
@@ -5,48 +5,54 @@
 // An array of basic colors used accross different lighting profiles
 // static const uint32_t colorPalette[] = {0xFF0000, 0xF0F00, 0x00F00, 0x00F0F,
 // 0x0000F, 0xF000F, 0x50F0F};
-static const uint32_t colorPalette[] = {0x9c0000, 0x9c9900, 0x1f9c00, 0x00979c,
-                                        0x003e9c, 0x39009c, 0x9c008f};
+static const uint32_t colorPalette[] = {0xcc0000, 0xccc900, 0x5fcc00, 0x00c7cc,
```

Comment:
`colorPalette` is used in code that is called often as `... % LEN(colorPalette)`. Ensuring the size of the color palette is a power of 2 significantly improves performance of animation callbacks.
In addition, I increased color intensities because lower intensity values make the flicker more noticeable.

Reply:
+1